status.py

```
@@ -3,69 +3,95 @@
+CURRENT_BRANCH_PATH = os.path.join(directory_path, ".sccs", "current_branch", "current_branch.json")
 
-from sccs_layout_check import path
+def get_current_branch():
+    try:
+        with open(CURRENT_BRANCH_PATH, "r", encoding="utf-8", newline="\n") as current_branch_file:
+            try:
+                current_branch = json.load(current_branch_file).get("current_branch")
+            except json.JSONDecodeError as e:
+                print(f"Error decoding JSON from current branch file: {e}")
+                sys.exit(1)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)